jobs/JGLOBAL_WAVE_POST_BNDPNTBLL

```diff
@@ -13,9 +13,15 @@ export CDATE=${PDY}${cyc}
 export MP_PULSE=0
 
 # Set COM Paths and GETGES environment
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP COM_WAVE_HISTORY COM_WAVE_STATION
-
-if [[ ! -d ${COM_WAVE_STATION} ]]; then mkdir -p "${COM_WAVE_STATION}"; fi
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+   COMIN_WAVE_PREP:COM_WAVE_PREP_TMPL \
+   COMIN_WAVE_STATION:COM_WAVE_STATION_TMPL \
+   COMOUT_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL \
```

Comment:
Why is this a `COMOUT`? Is this job writing out to this directory? 
What are the inputs to this job?

Reply:
See https://github.com/HenryWinterbottom-NOAA/global-workflow/blob/526c64ccd997e316a6eba781fd14cacaf41ba897/jobs/JGLOBAL_WAVE_POST_BNDPNTBLL#L22.

Reply:
I just did a quick check and I don't see anywhere that we're writing out to the wave history directory - although it is a bit odd considering the line for creating the directory.  I assume that's a mistake from copy/pasting from something else and was not properly cleaned up.

Reply:
Thank you for the clarification, @JessicaMeixner-NOAA. In that case, all `COMOUT` should then be `COMIN` as noted by @aerorahul. Correct?

Reply:
I think COMOUT_WAVE_HISTORY -> COMIN_WAVE_HISTORY 

I'm checking on COMIN_WAVE_STATION I think this should be COMOUT... but i'll have to look .

Reply:
> I think COMOUT_WAVE_HISTORY -> COMIN_WAVE_HISTORY

@aerorahul Do you agree?

Reply:
Yes, WAVE_STATION is where output goes: https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/wave_tar.sh#L185 

So COMIN_WAVE_STATION -> COMOUT_WAVE_STATION 

WAVE_PREP and WAVE_HISTORY are where inputs are coming from.

Reply:
Thanks @JessicaMeixner-NOAA 
That was also my understanding.

Reply:
Done.

Reply:
@WalterKolczynski-NOAA Can you please confirm if the suggestions, above, are valid? These seem to conflict with https://github.com/HenryWinterbottom-NOAA/global-workflow/blob/feature/gwdev_issue_2451/ush/forecast_postdet.sh

Reply:
@HenryWinterbottom-NOAA the forecast model writes to this location, this post job reads from this location.